_includes/payment-order-other-features.md

```diff
@@ -647,7 +647,7 @@ Content-Type: application/json
 | └➔&nbsp;`remainingCancelAmount`   | `integer`    | The available amount to cancel.                                                                                                                                                                                                                                                                                                                            |
 | └➔&nbsp;`remainingReversalAmount` | `integer`    | The available amount to reverse.                                                                                                                                                                                                                                                                                                                           |
 | └➔&nbsp;`description`             | `string(40)` | {% include field-description-description.md documentation_section=include.documentation_section %}                                                                                                                                                                                                                                                         |
-| └➔&nbsp;`payerReference`          | `string`     | The reference to the payer from the merchant system, like e-mail address, mobile number, customer number etc.                                                                                                                                                                                                                          |
+| └➔&nbsp;`payerReference`          | `string`     | The reference to the payer from the merchant system, like e-mail address, mobile number, customer number etc. Also used in [Payer Aware Payment Menu](#payer-aware-payment-menu).                                                                                                                                                                                                                          |
```

Comment:
I think we need to make this conditional on the `documentation_section`, as we don't want to cross-reference Checkout and Payment Menu.